packages/checkout-storefront/package.json

```
@@ -11,7 +11,9 @@
     "generate": "DOTENV_CONFIG_PATH=.env graphql-codegen -r env-vars --config codegen.yml",
     "check-types": "tsc --noEmit",
     "lint": "eslint --max-warnings=0 \"src/**/*.@(tsx|ts|jsx|js)\" --fix",
-    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist"
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
+    "test": "jest",
+    "test-dev": "jest --watch"
```

Can we use that instead? I think we have this pattern in main `package.json`, so might as well use it here as well :P
```suggestion
    "test:dev": "jest --watch"
```

---

Okay